tests/container/BUILD

```diff
@@ -334,6 +334,44 @@ container_push(
     tags = ["manual"],
 )
 
+genrule(
```

Comment:
Is this the way you intend to use this in your repository (via `genrule`)? Could we add an example of how this might actually be used as a test? That would make providing support for this feature easier as we could link to the test for a demo of the approximate use case.

Reply:
I think we would likely leverage a custom `rule` (or `repository_rule`) for creating the inputs for `registry_file/repository_file`, as I'd expect those to be conditional on 1) Where are we working in? (sandbox/staging/etc), and 2) Which entity/user does this repository belong to? (e.g. `myuser/mycontainer`). 

The `genrule` is a stand-in for the command execution that would be responsible for determining the registry based on those conditions (1 & 2).